CONTRIBUTING.md

```diff
@@ -77,7 +91,15 @@ SwiftNIO has been created to be high performance.  The integration tests cover s
 
 Try to keep your lines less than 120 characters long so github can correctly display your changes.
 
-It is intended SwiftNIO will use the swift-format tool in the future to bring consistency to code formatting.  To follow the discussion on this topic see the swift evolution proposal [SE-250](https://github.com/apple/swift-evolution/blob/main/proposals/0250-swift-style-guide-and-formatter.md)
+SwiftNIO now uses the [swift-format](https://github.com/swiftlang/swift-format) tool to bring consistency to code formatting.    Their is a specific [.swift-format](./.swift-format) configuration file.    This will be checked and enforced on PRs, note that the check will run on the current most recent stable version target which may not match that in your own local development environment.
```

Comment:
```suggestion
SwiftNIO uses the [swift-format](https://github.com/swiftlang/swift-format) tool to bring consistency to code formatting.  There is a specific [.swift-format](./.swift-format) configuration file.  This will be checked and enforced on PRs.  Note that the check will run on the current most recent stable version target which may not match that in your own local development environment.
```